tuxemon/core/components/event/actions/combat.py

```diff
@@ -237,6 +237,9 @@ def random_encounter(self, game, action):
         # If a random encounter was successfully rolled, look up the monster and start the
         # battle.
         if encounter:
+            #stop player motion
+            player1.moving = False
```

Comment:
This line should be removed too if we're going to address #146 later. Otherwise the player will still become offset from the tile grid.